components/docs-chef-io/content/automate/supermarket_with_automate_ha.md

```diff
@@ -0,0 +1,113 @@
+    weight = 75
++++
+
+This guide will show you how to register supermarket with Automate HA embedded OC-ID. Before we start with this page, refer to the [Integration Supermarket](/automate/supermarket_integration_with_automate/) page with Automate.
+
+## Register Supermarket with Automate HA Embedded OC-ID
+
+The steps to register supermarket with Automate HA embedded OC-ID is as follows:
+
+1. SSH into the Bastion node of Automate HA On-Premises setup.
+
+    ```bash
+    sudo su
+    ```
+
+1. Create file named `ocid-apps.toml`, mention the **name** and the **redirect_uri** for the file. The content of the created file looks like as shown below:
+
+    ```cd
+    [ocid.v1.sys.ocid.oauth_application_config]
+        [[ocid.v1.sys.ocid.oauth_application_config.oauth_applications]]
+            name = ""
+            redirect_uri = ""
+    ```
+
+    Update the URL of the supermarket website in the `redirect_uri` as per the actual supermarket URL. efe to the code below:
+
+    ```cd
+    [ocid.v1.sys.ocid.oauth_application_config]
+        [[ocid.v1.sys.ocid.oauth_application_config.oauth_applications]]
+            name = "supermarket"
+            redirect_uri = "https://ec2-3-135-208-2.us-east-2.compute.amazonaws.com/auth/chef_oauth2/callback"
+    ```
+
+1. Now, run run `config patch` on all the automate and chef-server nodes from the bastion node. The commands to patch are as follows:
+
+    ```bash
+    # automate patch
+    chef-automate config patch ocid-apps.toml -a
+    ```
+
+    ```bash
+    # chef-server patch
+    chef-automate config patch ocid-apps.toml -c
+    ```
+
+    Once the patch is successfully completed the new app should be registered on OCID.
+
+1. Get the details of the registered applications from the bastion node, run the following command:
+
+    ```bash
+    chef-automate config oc-id-show-app -a
+    ```
+
+    The above command will fetch the details of all the registered OC-ID applications from all the Automate nodes. To know the flag usage for this command, run the following command: 
+
+    ```bash
+    chef-automate config oc-id-show-app -h.
+    ```
+
+    The output of the above command looks like as shown below:
+
+    IMAGE
+
+1. Now, SSH into the supermarket instance using the following command:
+
+    ```bash
+    sudo su
+    cd /etc/supermaket
+    ```
+
+1. Once you are in the path `/etc/supermarket`, edit the file `supermarket.rb` to update the details of the **UID**, **secret**, **chef-server-endpoint** and **ssl-verify-mode** as per the details of the registered OC-ID apps found in the steps above. Refer to the snippet below:
+
+    ```bash
+    default['supermarket']['chef_oauth2_app_id'] = "<UID>"
+    default['supermarket']['chef_oauth2_secret'] = "<Secret>"
+    default['supermarket']['chef_oauth2_url'] = "<Automate HA load balancer FQDN>"
+    default['supermarket']['chef_oauth2_verify_ssl'] = false
+    ```
+
```

Comment:
```suggestion

    N.B. The datatype of the attribute: `chef_oauth2_verify_ssl` is a boolean. The value will depend on where your automate is running with a valid SSL certificate or not. If automate is running with a valid SSL certificate then this should be set as `true` otherwise set is as `false`.

```